.github/workflows/ci.yml

```diff
@@ -228,7 +228,7 @@ jobs:
         submodules: true
     - name: start docker
       run: |
-        docker run -w /src -dit --name alpine -v $PWD:/src node:lts-alpine
+        docker run -w /src -dit linux/arm64 --name alpine -v $PWD:/src node:lts-alpine
```

Comment:
do we need docker here?

Reply:
I think that is how we get alpine linux .. otherwise we will be building for ubuntu